src/utils/hact/workOrderSchedule/raiseWorkOrderForm.test.js

```diff
@@ -162,4 +188,95 @@ describe('buildScheduleWorkOrderFormData', () => {
     const response = buildScheduleWorkOrderFormData(workOrderData)
     expect(response).toEqual(scheduleWorkOrderFormData)
   })
+
+  it('builds the ScheduleWorkOrder form data to post to the Repairs API with Planned Priority', async () => {
```

Comment:
👍  This is a good test to add, which I didn't think of. However we will still benefit from a test at the unit level in this file `completionDateTimes.test.js`. Sorry - I wasn't clear enough in my earlier comment.

You can just add another `describe` at the top level e.g. `'when the plannedPriority is true'`. I think it's important that these target times are put through their paces so having the additional coverage should be worth it.